content/docs/cve/cveCreateSession.md

```diff
@@ -46,29 +46,32 @@ Detailed instructions to schedule an exam session in the ExamTools software are
+  * Do not list on HamStudy.org – Hide your session and do not display on HamStudy.  Especially helpful, if you want to work on your session listing ahead of time and aren’t quite ready for applicants or want to share a private link to a certain population for whom the session has been scheduled.  (i.e. Local radio club or college course)  The session is still accessible online, but the applicant will need a private link to view.  When you are ready to list your session on HamStudy, return to the session in ET and uncheck the box and it will appear in the HamStudy list of [Upcoming test sessions](https://ham.study/sessions). 
+  * Receive notification on applicant signup - If this box is selected, a New Applicant Registration notice will be sent to the email listed in the Contact Name and Email field. This email will include the following applicant information: Name, License Class, FRN, Address, Email, Phone Number and the date/time of the session in which applicant registered.
 
-* You may need to toggle the ExamTools Features Desired back to Choose… in order to add or remove the check mark next to not list with Schedule only.  You will see the following, if your listing is set to not display.  
+* Toggle the ExamTools Features Desired dropdown back to Choose… in order to add or remove a check mark for Schedule only and edit the options above.  You will see the Schedule only description in the blue box, if your listing is set to not display.  In that case, you will need to go back to Choose... to remove the Do not list on HamStudy.org selection.
 
   * **Schedule only** - Your exam will not appear on the schedule and online registration will be disabled.
 
-  You may want to do this to see the posting online, but for some reason do not want the general public to view it.  You may do this to proof-read it, experiment with formatting or share it with your VE team before it goes live.
+  Select the Schedule only and Do not list on HamStudy.org combo when you do not want the public to view it and register.  You may do this to proof-read it, experiment with formatting or share it with your VE team before it goes live using the private link.
+  
+  * **Contact Name and Email** - This field will include the Session Manager's email address by default, but may be changed to a preferred email address as long as the formatting is maintained.  ET uses this email for notification and does not appear in your session listing online.  If you want your email to be posted, added it to the Other Notes below.
```

Comment:
> ET uses this email for notification and does not appear in your session listing online.

ET uses this email for notification and it does not appear in your session listing online.

(added `it`)